### DISCUSSION TITLE
Add ability to specify queue name for ActiveJob

### DISCUSSION THREAD

**Discussion by k0va1** [pay-rails/pay#1025] [Ideas]
Pay jobs are always queued to `default` queue at the moment. I've encountered with a long delay in webhooks processing in case when there're a lot of jobs in this queue. It would be great to have an option to specify queue name in pay configuration to handle jobs priority

**Comment by excid3** [comment]
You can customize anything you want by including a module in an initializer.

```ruby
module PayJobQueue
  extend ActiveSupport::Concern

  included do
    queue_as :low_priority
  end
end

Pay::ApplicationJob.include PayJobQueue
```
